docs/docs/build/examples/btc_swap/index.md

```diff
@@ -1,34 +1,170 @@
 # BTC Swap
 
-This example demonstrates how BOB can be used to build a peer to peer swap application which allows BTC to be exchanged for an ERC20 token without the use of a custodian.
+This example demonstrates how BOB is leveraged to build a peer-to-peer (P2P) swap application that allows two parties to swap ERC20 tokens and BTC without using a custodian.
 
 :::note
 
-This example app is a work in progress, and the example application repository is currently set to private.
+The code for this example can be [found in the P2P swap demo GitHub repository](https://github.com/bob-collective/p2p-swap-demo)
 
 :::
 
-## The contract
+## Demo
+
+We have created a testnet demo of the P2P swap, allowing anyone to swap BTC for ERC20 tokens, ERC20 tokens for BTC, and exchange one ERC20 token for another. The application currently supports testnet BTC, testnet USDT, and ZBTC (ZBTC is a placeholder for an arbitrary bridged BTC token).
+
+### Connecting Your Wallet
+
+:::note
+
+When connecting to our test network the gas token will be shown as BOB. This can be changed to ETH in the network settings of your wallet.
```

Comment:
Doesn't that depend on how you add the network to the wallet? By default, the network doesn't assume any token.

Reply:
```suggestion
When connecting to our test network the gas token ticker may be shown as something other then ETH. This can be changed to ETH in the network settings of your wallet.
```

Reply:
Does that make more sense? If the network is added through Conduit it shows a field to set the ticker, but if you connect with our 'Connect Wallet' button it defaults to Bob, and you have to change it in the settings (in Metamask, at least).